src/Protocol/ActivityPub/Transmitter.php

```diff
@@ -62,72 +62,26 @@
 class Transmitter
 {
 	/**
-	 * collects the lost of followers of the given owner
+	 * Collects a list of contacts of the given owner
 	 *
-	 * @param array   $owner Owner array
-	 * @param integer $page  Page number
+	 * @param array     $owner  Owner array
+	 * @param int|array $rel    The relevant value(s) contact.rel should match
+	 * @param string    $module The name of the relevant AP endpoint module (followers|following)
+	 * @param integer   $page   Page number
 	 *
 	 * @return array of owners
-	 * @throws \Friendica\Network\HTTPException\InternalServerErrorException
+	 * @throws \Exception
 	 */
-	public static function getFollowers($owner, $page = null)
+	public static function getContacts($owner, $rel, $module, $page = null)
 	{
-		$condition = ['rel' => [Contact::FOLLOWER, Contact::FRIEND], 'network' => Protocol::FEDERATED, 'uid' => $owner['uid'],
+		$condition = ['rel' => $rel, 'network' => Protocol::ACTIVITYPUB, 'uid' => $owner['uid'],
```

Comment:
We mustn't query the contacts this way, since this would filter out most Friendica systems and (partially) systems like Hubzilla or Socialhome. If we want to publish only AP enabled contacts, we have to check against the `apcontact` table.

Reply:
I was hoping you would chime in, I knew my solution would be too restrictive but I wasn't sure what was the right way. Thanks for the pointer.